css/components/list.css

```diff
@@ -0,0 +1,41 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700;800&display=swap');
+
+@import url(../colorVariables.css);
+
+h1{
+    font-family: 'Poppins', sans-serif;
+}
+  
+.list-spaced{
+    font-family: 'Poppins', sans-serif;
+    list-style: none;
+    padding-left: 0;
+    display: flex;
+    flex-direction: row;
+}
+
+.list-spaced li{
+    background-color: var(--listItemBG);
+    margin: 0.5rem;
+    padding: 0.5rem 1.2rem;
+    border-radius: 0.3rem;
+    font-weight: 500;
+    border: 2px solid var(--zinc);
+}
+
+.list-stacked{
+    font-family: 'Poppins', sans-serif;
```

Comment:
Since you are using this font at multiple places you can create a variable for it as well.